README.md

```diff
@@ -1,10 +1,8 @@
-# flutter-code-editor
+# Flutter Code Editor
 
-A multi-platform code editor supporting
-syntax highlighting,
-code folding,
-autocompletion,
-and partial locking.
+Flutter Code Editor is a multi-platform code editor supporting syntax highlighting, code blocks folding, autocompletion, named sections, read-only code blocks, hiding text to create code snippets, themes and many other features.
```

Comment:
Better not list named sections here because this is a service feature of no use by itself. It is only a mean to implement other features.